src/main/java/de/zalando/zmon/scheduler/ng/CommandSerializer.java

```diff
@@ -28,6 +25,19 @@ public String expiresTime(long interval) {
         return LocalDateFormatter.get().format(exp);
     }
 
+    public byte[] writeCleanUp(Set<String> removedIds){
+        CeleryBody body = new CeleryBody();
+
+        body.task= "cleanup";
```

Comment:
I know that we don't have a linter for java projects yet, but what do you think if we'll try to follow some basic formatting rules, common for most languages? Like spaces before and after `=` and `+` for example.

Reply:
agreed